osm2streets-js/src/lib.rs

```diff
@@ -0,0 +1,29 @@
+use serde::{Deserialize, Serialize};
+use wasm_bindgen::prelude::*;
+
+// TODO Placeholder
+
+#[derive(Serialize, Deserialize)]
+pub struct Input {
+    drive_on_right: bool,
+}
+
+#[wasm_bindgen]
+pub fn import_osm(val: &JsValue) -> String {
```

Comment:
Next step here will be taking a huge osm.xml blob result from Overpass as input, and returning the graphviz and geometry layers as output. Or... maybe the Overpass query should live in Rust-land like osm2lanes, in which case the input will just be a bounding box. Either way, that's my next step

Reply:
My initial impression is that the fetching should be done in JS, because `fetch` and `await` make it very erganomic, and I would want the osm2lanes JS API to accept an osm.xml blob/string as an input for flexibility.

That being said, if there was another function exposed by the JS API that lets me specify a bounding box and osm2lanes does the loading behind the scenes, that would be cool too!

Reply:
That's the approach I went with in #46. The argument for putting it in Rust would be to share more code if/when we have Java, Python, etc callers. But it's definitely much easier to write this in JS than pulling in `reqwest` dependencies.

Reply:
Yeah, using overpass servers for data loading sounds more like a decision that the user of the lib should be making (even if we make it super easy).

Reply:
Right, Overpass isn't the only option, and it won't be appropriate for dynamically loading large areas. Sounds like it's good to keep this at the JS layer.